src/ui/components/Rating/styles.scss

```diff
@@ -55,23 +55,19 @@
 }
 
 .Rating--small .Rating-choice,
-.Rating--small-monochrome .Rating-choice {
+.Rating--small-by-user .Rating-choice {
```

Comment:
I don't see `Rating--small-by-user` used anywhere -- is that still necessary? If so, it should be `...by-owner` for consistency.